RIOT-OS/RIOT
# tests/spiffs: Move from unittests to regular test

### Contribution description

another one related to #10187

### Testing procedure

Verify that `tests/pkg_spiffs` works: `BOARD=<your-favourite> make -C tests/pkg_spiffs flash test`

### Issues/PRs references

#10187
